### ISSUE TITLE
Website Servicing

### ISSUE THREAD

**Issue by ElizabethP712** [emiliom/squid-dinosip#13]
# Website Down for Servicing Question

I had a quick question after running into the issue today. It turns out the erddap website is being serviced so I cannot run my code to check it works while uploading it to this repo. Is there a place on the internet they estimate how long this servicing will take?

https://erddap.ifremer.fr/erddap/tabledap/ArgoFloats.html

**Comment by emiliom** [comment]
Hmm. I looked around the parent websites and couldn't find any notification about the status of https://erddap.ifremer.fr. I suggest checking back in an hour, late tonight or early tomorrow morning. There may be some maintenance going on; but since this is a European server and it's 12:30am there, I'm guessing that  if it's an automated maintenance step it'll either be back shortly, or if it's an unexpected issue, they won't get to it until their morning.

**Comment by ElizabethP712** [comment]
Hey Emilio!

I am still having a hard time with this website and I sort of need it to finish all my code. Do you know who I should contact to report an error/issue with the erddap Argo data?

**Comment by emiliom** [comment]
I doubt an error report would result in a fix in time for you to benefit from it. I'd be very surprised if they're not already aware of the problem.

Try using `src='argovis'` (see https://argopy.readthedocs.io/en/latest/user-guide/fetching-argo-data/data_sources.html). The results may be a bit different than what you've been using (`src='erddap'`), though.

**Comment by ElizabethP712** [comment]
I will try it out again, but I was running into some of the same access issues that drove us to use erddap in the first place. I will probably end up revisiting the code sometime early September just to wrap things up. For now, I can still compile overview documents with written instructions, comments, etc. and then run them later and work through any issues. 

It looks like another option could also be 'gdac' too, which works for some of my IndexFetcher code, but a lot of the graphing code is still not running without errdap. Thanks for the suggestion though!

**Comment by emiliom** [comment]
Yeah, it does look like you won't be able to rely on this in time. Keep going as you describe, and we can discuss on Thursday, if there's time.
